pypeit/par/pypeitpar.py

```diff
@@ -3325,6 +3325,13 @@ def __init__(self, filt_iter=None, sobel_mode=None, edge_thresh=None, sobel_enha
                                 'image (see `trace_median_frac`), values in the median-filtered ' \
                                 'image *below* this threshold are masked in the refitting of ' \
                                 'the edge trace data.  If None, no masking applied.'
+        
+        dtypes['trace_rms_tol'] = [int, float]
+        descr['trace_rms_tol'] = 'After retracing edges using peaks detected in the rectified ' \
+                                 'and collapsed image, the RMS difference between the original ' \
+                                 'and refit traces are calculated.  This sets the upper limit ' \
```

Comment:
It's probably obvious, but maybe just be explicitly clear that the units is pixels

Reply:
Good catch.  Added!